File: nan21.dnet.module.ad/nan21.dnet.module.ad.presenter/src/main/java/net/nan21/dnet/module/ad/usr/ds/model/UserGroupLovDs.java
/*
 * DNet eBusiness Suite
 * Copyright: 2008-2012 Nan21 Electronics SRL. All rights reserved.
 * Use is subject to license terms. 
 */
package net.nan21.dnet.module.ad.usr.ds.model;

import net.nan21.dnet.core.api.annotation.Ds;
import net.nan21.dnet.core.api.annotation.SortField;
import net.nan21.dnet.core.presenter.model.base.AbstractTypeLov;
import net.nan21.dnet.module.ad.usr.domain.entity.UserGroup;

@Ds(entity = UserGroup.class, jpqlWhere = " e.active = true ", sort = { @SortField(field = UserGroupLovDs.fNAME) })
public class UserGroupLovDs extends AbstractTypeLov<UserGroup> {

    public UserGroupLovDs() {
        super();
    }

    public UserGroupLovDs(UserGroup e) {
        super(e);
    }

}

File: nan21.dnet.module.ad/nan21.dnet.module.ad.ui.extjs/src/main/webapp/src/ds/UserGroupLovDs.js
 
   	 
Ext.define("net.nan21.dnet.module.ad.usr.ds.model.UserGroupLovDs" , {
	extend: 'Ext.data.Model',
	fields: [  		
		{name:"id",type:"int", useNull:true }, 		
		{name:"clientId",type:"int", useNull:true }, 		
		{name:"name",type:"string" }, 		
		{name:"active",type:"boolean" }
	]
});
	
Ext.define("net.nan21.dnet.module.ad.usr.ds.param.UserGroupLovDsParam", {
	extend: 'Ext.data.Model',
	fields: [ 
	]
}); 
 